src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -30,6 +34,7 @@ public User register(String email, String password) throws RegistrationException
             user.setEmail(email);
             user.setPassword(password);
             userService.add(user);
+            shoppingCartService.registerNewShoppingCart(user);
```

Comment:
it would be a better choice, In my opinion
```suggestion
            shoppingCartService.registerNewShoppingCart(userService.add(user));
```